bokeh/settings.py

```diff
@@ -326,6 +348,12 @@ def nodejs_path(self, default=None):
 #:   to an affirmative value will result in more friendly simple numeric IDs
 #:   counting up from 1000.
 #:
+#: ``BOKEH_USE_BINARY_ARRAYS`` --- Whether to encode data arrays before sending them.
+#:
+#:   Accepted values are ``yes``/``no``, ``true``/``false`` or ``0``/``1``.
+#:   Can have a huge effect on performance because encoding and decoding
+#:   arrays as raw strings can be very slow.
```

Comment:
My thought was that sometimes its nice to see the list output for debugging. However, I seem to be in the minority, so I would not oppose removing these.  I do agree and was thinking yesterday about the nan/inf issue.